cmd/pics/state.go

```diff
@@ -109,11 +108,73 @@ func keyTape(t *trie.Trie, number int) error {
 	return nil
 }
 
+func stateDatabaseMap(db* bolt.DB, number int) error {
+	filename := fmt.Sprintf("state_%d.dot", number)
+	f, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	i := 0
+	prevName := ""
+	clusterData := ""
```

Comment:
I suggest using `Builder` type from the `strings` package to accumulate clusterData, so that instead of
````
clusterData = fmt.Sprintg("%s ...", clusterData, ...)
````
 you use
````
clusterData.WriteString(fmt.Sprintf("....", ....))